RDF-Pipeline/t/accept-test.perl

```diff
@@ -52,6 +52,13 @@
 	my $copyCmd = "$moduleDir/t/helpers/copy-dir.perl '$tmpTDir' '$dir/expected-files'";
 	# warn "copyCmd: $copyCmd\n";
 	!system($copyCmd) or die;
+
+	# Tar the expected files and delete the expected-files directory.
+	# We use tar to ensure empty directories are handled correctly.  Git
+	# currently ignores empty directories .
+	!system("(cd $dir/expected-files; tar -czf ../expected-files.tar.gz .)") or die "Unable to create expected-files.tar.gz\n";
+	!system("rm -rf $dir/expected-files") or die "Unable to remove expected-files dir\n";
```

Comment:
I think we could now skip the creation of the expected-files directory entirely, and just generate expected-files.tar.gz directly from $tmpTDir, which is defined around line 108.